### PR TITLE
Add ChronosPlayer.app v1.2

### PR DESCRIPTION
**Pull request by nisar-med**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/chronosplayer.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'chronosplayer' do
+  version '1.4.110'
+  sha256 '9cab46861d52ad0f0fe0e67705fb382e0dc23752851a61f0724a9378558eded6'
+
+  url 'http://chronosplayer.s3-website-us-east-1.amazonaws.com/ChronosPlayer.zip'
+  name 'Chronos Player'
+  homepage 'http://chronosplayer.s3-website-us-east-1.amazonaws.com/'
```
Comment (commitay) [inline Casks/chronosplayer.rb]: Needs to be a working homepage, this URL 404s

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/chronosplayer.rb (hunk 1 above)
Comment (nisar-med) [reply Casks/chronosplayer.rb]: I will put a homepage ASAP

Comment (nisar-med) [reply Casks/chronosplayer.rb]: I have put a coming soon homepage on the URL.

**Comment by commitay** [comment]
Seems to be broken on 10.13

chronos

**Comment by commitay** [comment]
As this is your own app I'd suggest [creating your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md) for now.

Thanks for the PR nonetheless!

**Pull request closed without merging**